src/app/pages/dashboard/services/get-default-widgets.ts

```diff
@@ -54,3 +60,11 @@ export const defaultWidgets: WidgetGroup[] = [
     ],
   },
 ];
+
+export const getDefaultWidgets = (isHaLicensed?: boolean): WidgetGroup[] => {
+  if (!isHaLicensed) {
+    return defaultWidgets.filter((_, index) => index !== 1);
```

Comment:
That's fragile. Some time later someone will change the order of widgets and this code will stop working.

Reply:
> That's fragile. Some time later someone will change the order of widgets and this code will stop working.

@undsoft - yeah, I can override it. But tests will fail if someone change the order as well.